src/Components/cards/card.jsx

```diff
@@ -32,18 +32,28 @@ function ProductCard() {
               <b class="discount-price">24% OFF</b>
             </div>
             <div class="card-button">
-              <button
-               onClick={()=>dispatch({type:"ADD_TO_CART",payload:items})}
+            {cart.filter((item_) => item_._id == item._id).length ? <button
+               onClick={()=>{
+                 
+              }}
+                class="button-container-button primary-button cart"
+              >GO TO CART</button>  : <button
+               onClick={()=>{
+                AddToCart(item,dispatch)  
+              }}
                 class="button-container-button primary-button cart"
-              >
-                Add To Cart
-              </button>
+              >ADD TO CART</button> }
+              
+                
+              
             </div>
             <button class="exit-button">
               <span>
                 <img
```

Comment:
You can use icons instead of image can use react-icons from bootstrap or any other sources